src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((item) => item.trim() !== '')
+    .reduce((stylesObject, item) => {
+      const pair = item.split(':');
+
+      const key = pair[0].trim();
+      const value = pair[1].trim();
+
+      stylesObject[key] = value;
```

Comment:
It's good practice to check if `pair[1]` exists before attempting to call `.trim()` on it to prevent potential runtime errors if the CSS string is malformed.